src/main.rs

```diff
@@ -144,6 +146,15 @@ pub fn fmt_duration(style: DurationStyle, secs: i64) -> String {
                 format!("{}", s)
             }
         },
+        DurationStyle::MS => {
+            let m = secs / 60;
+            let s = secs % 60;
+            if m > 0 {
+                format!("{}:{:02}", m, s)
+            } else {
+                format!("0:{:02}", s)
+            }
+        },
```

Comment:
Both `if` branches output the same thing, just use the body of the first.

I find it a bit strange that you're not formatting hours. Merge times over 60 minutes are not uncommon, and displaying them as `123:45` doesn't seem very usable ? My idea of the new format was that it'd be the same as `Duration::HMS` but without ignoring leading zeros, so `format!("{}:{:02}:{:02}")` in all cases.